README.md

```diff
@@ -2,8 +2,38 @@
 
 > Shareable ESLint configuration used at d-centralize.
 
+[![npm](https://img.shields.io/npm/v/eslint-config-d-centralize.svg)](https://www.npmjs.com/package/eslint-config-d-centralize)
 [![Build Status](https://img.shields.io/travis/dcentralize/eslint-config-d-centralize/master.svg)](https://travis-ci.org/dcentralize/eslint-config-d-centralize)
 [![devDependency Status](https://img.shields.io/david/dev/dcentralize/eslint-config-d-centralize.svg)](https://david-dm.org/dcentralize/eslint-config-d-centralize/?type=dev)
-[![devDependency Status](https://img.shields.io/david/peer/dcentralize/eslint-config-d-centralize.svg)](https://david-dm.org/dcentralize/eslint-config-d-centralize/?type=peer)
+[![peerDependency Status](https://img.shields.io/david/peer/dcentralize/eslint-config-d-centralize.svg)](https://david-dm.org/dcentralize/eslint-config-d-centralize/?type=peer)
 
-This is still an early draft. Any changes may be breaking until version 1.0.0 is published to npm.
+This is a very opinionated shareable ESLint configuration. The goal of this project is to enforce a strict coding style at [d-centralize](http://d-centralize.nl) projects. This project is kept up-to-date with the latest and greatest ESLint [rules](http://eslint.org/docs/rules).
```

Comment:
I'd leave out the part where we say it's kept up to date. As observed from the version bump, we're already lagging for weeks ;-)